supernode/server/0.3_bridge.go

```diff
@@ -121,9 +121,11 @@ func (s *Server) pullPieceTask(ctx context.Context, rw http.ResponseWriter, req
 	if !cutil.IsEmptyStr(dstCID) {
 		dstDfgetTask, err := s.DfgetTaskMgr.Get(ctx, dstCID, taskID)
 		if err != nil {
-			return err
+			logrus.Warnf("srcCID:%s, failed to get dfget task by dstCID(%s) and taskID(%s), err: %v",
```

Comment:
please output the log as the following format: logrus.Warnf("failed to ........")

Reply:
I think it may be better
```
if err!=nil{
     logrus.Warnf("failed to ......")
     return  err
}
request.DstPID=.....
```
WDYT?

Reply:
in the end,I want you to show me the detail error log.I recently work on this part

Reply:
Done